s3torchconnector/src/s3torchconnector/s3map_dataset.py

```diff
@@ -26,19 +26,25 @@ class S3MapDataset(torch.utils.data.Dataset):
     def __init__(
         self,
         region: str,
+        endpoint: str,
```

Comment:
In `S3IterableDataset`, this parameter has a default value (`""`). It should be consistent between these two classes, and be in the same order.

Reply:
I made the S3MapDataset class consistent with S3IterableDataset